src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingController.kt

```diff
@@ -102,4 +104,29 @@ class VurderingController(
         gjenbrukVilkårService.gjenbrukInngangsvilkårVurderinger(request.behandlingId, request.kopierBehandlingId)
         return Ressurs.success(vurderingService.hentEllerOpprettVurderinger(request.behandlingId))
     }
+
+    @PostMapping("gjenbruk-enkelt-vilkar")
+    fun gjenbrukEnkeltVilkår(
+        @RequestBody request: EnkeltVilkårForGjenbrukRequest,
+    ): Ressurs<VilkårsvurderingDto> {
+        val behandlingForGjenbruk = gjenbrukVilkårService.finnBehandlingerForGjenbruk(request.behandlingId).first()
+
+        tilgangService.validerTilgangTilBehandling(behandlingForGjenbruk.id, AuditLoggerEvent.ACCESS)
```

Comment:
```suggestion
```
Tror du er interessert i å sjekke update-tilgang her

Reply:
Fikser

Reply:
Ikke for å være kjip, men det gir vel faktisk mening at denne er ACCESS - for vi henter ut data fra denne behandlingen, men updater den andre..?

Reply:
Ja, jeg er enig. Jeg trekker tilbake forslaget mitt 😇